File: backend-C#/DoctorsOffice/DbModels/Facility.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace DoctorsOffice.DbModels
{
    public class Facility
    {
        public int Id { get; set; }
        [MaxLength(100)] 
        public string Name { get; set; }
        public string Street { get; set; }
        public string PostalCode { get; set; }
        public string City { get; set; }
        public string PhoneNumber { get; set; }
        
        [JsonIgnore]
        public ICollection<Doctor> Doctors { get; set; }
    }
}

File: backend-Java/src/main/java/pl/kacste/facility/Facility.java
package pl.kacste.facility;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.Fetch;
import org.hibernate.annotations.FetchMode;
import pl.kacste.doctor.Doctor;

import javax.persistence.*;
import java.util.Set;

import static org.hibernate.annotations.FetchMode.SELECT;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Facility {
    @Id
    @GeneratedValue
    private Long id;

    private String name;
    private String street;
    private String postalCode;
    private String city;
    private String phoneNumber;

    @OneToMany(mappedBy = "facility")
    @Fetch(value = SELECT)
    @JsonIgnore
    private Set<Doctor> doctors;
}
